cmake/Modules/GrPython.cmake

```diff
@@ -93,12 +93,12 @@ endmacro(GR_PYTHON_CHECK_MODULE)
 ########################################################################
 if(NOT DEFINED GR_PYTHON_DIR)
 execute_process(COMMAND ${PYTHON_EXECUTABLE} -c "
-import os
-import sys
-if os.name == 'posix':
-    print(os.path.join('lib', 'python' + sys.version[:3], 'dist-packages'))
-if os.name == 'nt':
-    print(os.path.join('Lib', 'site-packages'))
+try:
+    from site import getsitepackages
+    print(getsitepackages()[0])
```

Comment:
At least for me: This line returns the whole path, while the other one returns a relative path. I'm not sure it makes a difference to CMake, but I think it's better for the value of GR_PYTHON_DIR to be consistently set: full or relative, pick one & go with it.

Reply:
I am not sure which would be the better choice so that everything worked 
> in and out of prefixes #2019

maybe @djanderson can provide some insight.

Reply:
Yes I think that will do the trick.

Reply:
The prior code was relative, but from testing absolute paths work too. So, we can go either way. I like your solution to just remove the `prefix=""` part.

Reply:
Very well, then.